lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -58,12 +67,21 @@ public function init() {
 	}
 
 	/**
-	 * Get the list of fonts.
+	 * Get the list of registered fonts.
 	 *
 	 * @return array
 	 */
-	public function get_fonts() {
-		return self::$webfonts;
+	public function get_registered_fonts() {
+		return self::$registered_webfonts;
```

Comment:
Reference the variable as an instance member because that's what it is.

```suggestion
		return $this->registered_webfonts;
```